src/Rector/Class_/MessageSubscriberInterfaceToAttributeRector.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Symfony\Rector\Class_;
+
+use PhpParser\Node;
+use PhpParser\Node\Stmt\Class_;
+use PhpParser\Node\Stmt\ClassMethod;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\ValueObject\PhpVersionFeature;
+use Rector\Symfony\Helper\MessengerHelper;
+use Rector\Symfony\NodeAnalyzer\ClassAnalyzer;
+use Rector\Symfony\NodeManipulator\ClassManipulator;
+use Rector\VersionBonding\Contract\MinPhpVersionInterface;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see Rector\Symfony\Tests\Rector\Class_\MessageSubscriberInterfaceToAttributeRector\MessageSubscriberInterfaceToAttributeRectorTest
+ */
+final class MessageSubscriberInterfaceToAttributeRector extends AbstractRector implements MinPhpVersionInterface
+{
+    public function __construct(
+        private readonly MessengerHelper $messengerHelper,
+        private readonly ClassManipulator $classManipulator,
+        private readonly ClassAnalyzer $classAnalyzer
+    ) {
+    }
+
+    public function provideMinPhpVersion(): int
+    {
+        return PhpVersionFeature::ATTRIBUTES;
+    }
+
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition(
+            'Removes MessageSubscriberInterface and adds AsMessageHandler attribute to methods',
```

Comment:
I think its nice to keep the description the same style as `MessageHandlerInterfaceToAttributeRector` i think `MessageHandlerInterfaceToAttributeRector` is a bit nicer but you may decide